engine/examples/commandline/gon-config.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
do we need this file on the commandline example? anyway I feel like this example shouldn't even exist

Reply:
Whoops, that's a duplicate of `products/commandline/gon-config.json`. It was added before the refactoring